src/extra/calibration.py

```diff
@@ -958,6 +958,16 @@ def display_markdown_table(self, module_naming="modnum"):
 class ConditionsBase:
     calibration_types = {}  # For subclasses: {calibration: [parameter names]}
 
+    def _repr_markdown_(self):
+        attr_names = [f.name for f in fields(self)]
+        items = []
+        for n in attr_names:
+            if (value := getattr(self, n)) is not None:
+                if not isinstance(value, str):
+                    value = float(value)
```

Comment:
Do you want this representation to what can be seen in CalCat? I'm just wondering whether we should keep the semantically *correcter* types as they're in the `dataclass`.

Reply:
Thanks!